lib/jira.js

```diff
@@ -668,7 +668,51 @@ var JiraApi = exports.JiraApi = function(protocol, host, port, username, passwor
                 return;
             }
 
-            if (response.statusCode !== 200) {
+            if (response.statusCode / 100 | 0 != 2) {
```

Comment:
This check is made in two places. Could you extract it to a helper function like `is200ErrorCode`?